src/modules/sync/utils/getContentModel.ts

```diff
@@ -17,19 +27,20 @@ export const readContentModelFromFolder = async (folderName: string): Promise<Fi
 
   const snippets = JSON.parse(
     await fs.readFile(`${folderName}/${contentTypeSnippetsFileName}`, "utf8"),
-  ) as unknown as ReadonlyArray<
-    ContentTypeSnippetsSyncModel
-  >;
+  ) as ReadonlyArray<ContentTypeSnippetsSyncModel>;
   const taxonomyGroups = JSON.parse(
     await fs.readFile(`${folderName}/${taxonomiesFileName}`, "utf8"),
-  ) as unknown as ReadonlyArray<
-    TaxonomySyncModel
-  >;
+  ) as ReadonlyArray<TaxonomySyncModel>;
+
+  const webSpotlight = JSON.parse(
```

Comment:
maybe we can use zod for this? 👀 If you don't want to do that in this PR, maybe we can create a task for that :)

Reply:
I'll create a task to use zod to validate all the input files (both for sync and import)

Reply:
https://kontent-ai.atlassian.net/browse/DEVREL-1202